manifests/entry/duplicity.pp

```diff
@@ -1,15 +1,15 @@
 define backupninja::entry::duplicity (
   $ensure             = $backupninja::ensure,
```

Comment:
What type?

Reply:
There is no need to ask for each ocurrence